### PR TITLE
fix(webapp): Only allow one error filter at a time

### PR DESCRIPTION
**Pull request by nalanj**
## Describe your changes

Right now it's kind of odd that you can select both error states but on the server it only applies the first one chosen. This makes it where the UI only allows one error state to be chosen, which is more consistent with the API.

## Issue ticket number and link

https://linear.app/nango/issue/NAN-2197/connections-error-filter-shoudnt-be-a-multiselect

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

**Comment by nalanj** [comment]
Reverted a change from another branch I was working on that I missed here.

File: packages/webapp/src/pages/Connection/List.tsx [hunk 1]
```diff
@@ -150,6 +150,11 @@ export const ConnectionList: React.FC = () => {
         setSelectedIntegration(values);
     };
 
+    const handleFilterErrorChange = (values: string[]) => {
+        const newItems = values.filter((f) => !filterWithError.includes(f));
+        setFilterWithError(newItems);
+    };
```
Comment (bodinsamuel) [inline packages/webapp/src/pages/Connection/List.tsx:156]: nitpick: I was lazy to do a proper filter but if you filter it doesn't make sense to store an array anymore.

**Review by bodinsamuel — APPROVED** [review]
Thanks 🙏🏻

File: packages/webapp/src/pages/Connection/List.tsx (hunk 1 above)
Comment (nalanj) [reply packages/webapp/src/pages/Connection/List.tsx:156]: Fixed in e95d69d8

**Comment by nalanj** [comment]
@bodinsamuel mind giving this another peek to see if that was what you meant?

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**